bench/trust_token.cc

```
@@ -0,0 +1,347 @@
+}
+
+BENCHMARK_CAPTURE(BM_SpeedTrustTokenKeyGen, SpeedTrustTokenKeyGen experiment_v1,
+                  TRUST_TOKEN_experiment_v1());
+BENCHMARK_CAPTURE(BM_SpeedTrustTokenKeyGen,
+                  SpeedTrustTokenKeyGen experiment_v2_voprf,
+                  TRUST_TOKEN_experiment_v2_voprf());
+BENCHMARK_CAPTURE(BM_SpeedTrustTokenKeyGen,
+                  SpeedTrustTokenKeyGen experiment_v2_pmb,
+                  TRUST_TOKEN_experiment_v2_pmb());
+
+#define SPEED(step, method, batchsize)                                         \
```

Clang (14.0.6)

[warning] function-like macro 'SPEED' used; consider a 'constexpr' template function

(cppcoreguidelines-macro-usage)